bundle-workflow/dockerfiles/ci-runner.centos8.dockerfile

```diff
@@ -0,0 +1,81 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+
+# This is a docker image specifically for standardize the ci/cd environment
+# for both developers and ci/cd tools in OpenSearch / OpenSearch-Dashboards
+# Please read the README.md file for all the information before using this dockerfile
+
+
+FROM centos:8
```

Comment:
Why do we need 2 images? Both Centos and AL2?

Reply:
Read description, KNN still needs CentOS8 to build.
@jmazanec15 would confirm this as the glibc version is different on both.

Reply:
Missed it. Thanks!